users/migrations/0002_static_data.py

```diff
@@ -1,22 +1,24 @@
 from datetime import datetime
 
 from django.db import migrations, transaction
+from django.contrib.auth.hashers import make_password
 
 
 def create_initial_users(apps, schema_editor):
     Profile = apps.get_model('users', 'Profile')
     User = apps.get_model('auth', 'User')
     profile_data = [
-        ('Ofek', 'qaz[;.123', datetime(2020, 5, 17), '050-123456789'),
-        ('Danny', 'qaz[;.123', datetime(2020, 5, 17), '050-223456789'),
-        ('Rimon', 'qaz[;.123', datetime(2020, 5, 17), '050-323456789'),
-        ('Rotem', 'qaz[;.123', datetime(2020, 5, 17), '050-423456789'),
-        ('Saar', 'qaz[;.123', datetime(2020, 5, 17), '050-523456789'),
-        ('Tal', 'qaz[;.123', datetime(2020, 5, 17), '050-623456789'),
+        ('Ofek', 'ofekPassword', datetime(2020, 5, 17), '050-123456789'),
+        ('Danny', 'dannyPassword', datetime(2020, 5, 17), '050-223456789'),
+        ('Rimon', 'rimonPassword', datetime(2020, 5, 17), '050-323456789'),
+        ('Rotem', 'rotemPassword', datetime(2020, 5, 17), '050-423456789'),
+        ('Saar', 'saarPassword', datetime(2020, 5, 17), '050-523456789'),
+        ('Tal', 'talPassword', datetime(2020, 5, 17), '050-623456789'),
     ]
     with transaction.atomic():
         for username, password, dob, phone_number in profile_data:
-            user = User(username=username, password=password)
+            encrypted_pw = make_password(password)
+            user = User(username=username, password=encrypted_pw)
```

Comment:
I think instead of encrypting it here (it in any way saved as encrypted)
you can check it in the test with the method:
check_password('dannyPassword', user.password)

```suggestion
            user = User(username=username)
            user.set_password(password)
```

Reply:
I tried it before. When I did, I got the following error:
"'User' object has no attribute 'set_password'"